virt/virt-2-4-release-notes.adoc

```diff
@@ -83,13 +84,17 @@ the resource limits applied to CDI worker Pods.
 * The `virtctl` tool can now use a `DataVolume` when uploading virtual machine disks to the cluster. This helps prevent virtual machines
 from being inadvertently started before an upload has completed.
 
-//CNV-5421 - DataVolumes have been enhanced with conditions that make it easier to understand the state of VM disk import, clone, and upload.
+//CNV-5421 - OpenShift Container Storage (OCS) DataVolumes 
+* OpenShift Container Storage (OCS) DataVolumes have been enhanced with conditions and events that make it easier to understand the state of virtual disk imports, clones, and upload operations.
```

Comment:
I don't think you need `(OCS)` here if you add it to the previous release note (see my other comment).

Reply:
Done.